nomad/grapl-core.nomad

```diff
@@ -305,6 +314,14 @@ locals {
 
   # enabled
   rust_backtrace = 1
+
+  heredoc = <<EOH
+GRAPL_AWS_ENDPOINT          = ${local.aws_endpoint}
+GRAPL_AWS_ACCESS_KEY_ID     = ${var.aws_access_key_id}
+GRAPL_AWS_ACCESS_KEY_SECRET = ${var.aws_access_key_secret}
+EOH
+
+  local_vars = (var._aws_endpoint != "http://LOCAL_GRAPL_REPLACE_IP:4566") ? "IN_AWS=TRUE" : local.heredoc
```

Comment:
The template requires that some environment variable be defined. I originally was setting it to an empty string, which broke nomad's ability to parse its diff. If there's a different env var you want to be defined in AWS I'm open to suggestions